.circleci/config.yml

```diff
@@ -0,0 +1,53 @@
+version: 2.1
+
+orbs:
+  codacy: codacy/base@5.0.5
+  codacy_plugins_test: codacy/plugins-test@0.14.6
+
+workflows:
+  version: 2
+  compile_test_deploy:
+    jobs:
+      - codacy/checkout_and_version:
+          write_sbt_version: true
+      - codacy/sbt:
+          name: publish_docker_local
+          cmd: |
+            sbt "set scalafmtUseIvy in ThisBuild := false;
+                 scalafmt::test;
+                 sbt:scalafmt::test;
+                 set name := \"$CIRCLE_PROJECT_REPONAME\";
+                 universal:stage"
```

Comment:
It failed when I tried to use graalvm, so I'm going with stage right now. Do you have other suggestions?

Reply:
I'd say that it is advisable to remain on the JVM in tools that are written in Java, because they could use reflection under the hood, which is not supported by Graalvm native-image.